### ISSUE TITLE
It is not showing up in th e addons

### ISSUE THREAD

**Issue by gerroon** [ksami/QRemeshify#7]
Blender 4.3 Beta Windows 11, Qremeshify 1.0.2

I see that it throws this error during Blender startup

```
Warning: add-on missing 'bl_info', this can cause poor performance!: 'C:\\Users\\USER\\AppData\\Roaming\\Blender Foundation\\Blender\\4.3\\scripts\\addons\\QRemeshify\\__init__.py
```

![image](https://github.com/user-attachments/assets/256f9ad2-75b3-4531-9f94-367b52dca352)

**Comment by ksami** [comment]
I'm not sure what has changed in 4.3 as it's not released yet, moreover the [4.2 docs state to remove bl_info](https://docs.blender.org/manual/en/latest/advanced/extensions/addons.html#converting-a-legacy-add-on-into-an-extension). Either way, I've added a bl_info in v1.1.0 and it doesn't seem to affect anything in 4.2, please try it again and let me know

**Comment by gerroon** [comment]
Thanks for the reply, I found the issue. It shows up in the list if I put it under the user extensions folder. It doesn't work under the regular scripts folder.
